x-pack/test/api_integration/deployment_agnostic/apis/observability/apm/service_maps/service_maps.spec.ts

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { ApmSynthtraceEsClient } from '@kbn/apm-synthtrace';
+import expect from 'expect';
+import { serviceMap, timerange } from '@kbn/apm-synthtrace-client';
+import { Readable } from 'node:stream';
+import type { SupertestReturnType } from '../../../../../../apm_api_integration/common/apm_api_supertest';
+import type { DeploymentAgnosticFtrProviderContext } from '../../../../ftr_provider_context';
+
+type DependencyResponse = SupertestReturnType<'GET /internal/apm/service-map/dependency'>;
+type ServiceNodeResponse =
+  SupertestReturnType<'GET /internal/apm/service-map/service/{serviceName}'>;
+
+export default function ({ getService }: DeploymentAgnosticFtrProviderContext) {
+  const apmApiClient = getService('apmApi');
```

Comment:
Does this client need cleanup in the after method?

Reply:
Why is it not needed?  Just so I know for other reviews I'll have to do

Reply:
When this service was created there was no implementation of a`.destroy()` method. If you [take a look](https://github.com/elastic/kibana/issues/193245) at other PRs we have done, none of them have any clean up function for this client.
[Here](https://github.com/elastic/kibana/blob/main/x-pack/test/api_integration/deployment_agnostic/services/apm_api.ts) is the full code if you want to take a look